### PR TITLE
Vet Clinic Database

### PR DESCRIPTION
**Pull request by Rebelzob**
# Database requirements

- ### Added a column species of type string to the animal's table. 
1  create column

- ### Inserted the new values.
2  add data

- ### Inside a transaction updated the animal's table by setting the species column to unspecified. Verified that change was made. Then rolled back the change and verified that the species columns went back to the state before the transaction.
3  unspecified
4  species rollback

- ### Inside a transaction:
    >Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
    >Update the animal's table by setting the species column to pokemon for all animals that don't have species already set.
    >Verify that changes were made.
    >Commit the transaction.
    >Verify that changes persist after the commit.
5  update species

5  screenshot updated

- ### Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)

6  Delete rollback

- ### Inside a transaction:
    >Delete all animals born after Jan 1st, 2022.
    >Create a savepoint for the transaction.
    >Update all animals' weight to be their weight multiplied by -1.
    >Rollback to the savepoint
    >Update all animals' weights that are negative to be their weight multiplied by -1.
    >Commit transaction

7 3 savepoint

- ### Write queries to answer the following questions:
- How many animals are there?
8 1 how many animals

- How many animals have never tried to escape?
8 2 how many animals never escaped

- What is the average weight of animals?
8 3 average weight

- Who escapes the most, neutered or not neutered animals?
8 4 who escaped the most, neutered

- What is the minimum and maximum weight of each type of animal?
8 5 max min weight by specie

- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
8 6 escape average by birth date

### CODE REVIEW

**Review by omar-muhamad — COMMENTED** [review]
# Invalid ❌

Hi @Rebelzob,

#### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

#### Your Code Review Request is marked as invalid for this reason:

- [x] You didn't document your work professionally:
   - Your README file is missing an important section `Key Features`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ After you're finished please remove all the comments and instructions!
 
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
+    - [Key Features](#key-features)
   - [💻 Getting Started ](#-getting-started-)
     - [Prerequisites](#prerequisites)
     - [Setup](#setup)
```
Comment (thecodechaser) [inline README.md:34]: - [ ] The License link in the README file is broken, Please make sure to fix this. It will make the README file more professional.

Comment (thecodechaser) [inline README.md:34]: - [ ] You missed adding column `species` in the animals' table in the `schema.sql` file. Please make sure to add that as requested in the project requirements.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Rebelzob👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with transactions. :heavy_check_mark:
- Gitlow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Rebelzob, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**